src/components/Quiz/QuizWidget.tsx

```diff
@@ -0,0 +1,411 @@
+  return (
+    <Flex width="full" direction="column" alignItems="center">
+      <Heading
+        as="h2"
+        mb={12}
+        textAlign="center"
+        scrollBehavior="smooth"
+        scrollMarginTop={24}
+        id="quiz"
+      >
+        Test your knowledge
+      </Heading>
+      <Box
+        w="full"
+        maxW="600px"
+        bg={
+          !showAnswer
+            ? "neutral"
+            : currentQuestionAnswerChoice?.isCorrect
+            ? "successNeutral"
+            : "errorNeutral"
+        }
+        borderRadius="base"
+        boxShadow="drop"
+        pt={12}
+        pb={{ base: 5, md: 12 }}
+        px={{ base: 7, md: 15 }}
+        position="relative"
+        isolation="isolate"
+      >
+        {showConfetti && (
+          <>
+            <Icon
+              as={StarConfetti}
+              fontSize="184px"
+              position="absolute"
+              zIndex={-1}
+              top={-8}
+              left={0}
+            />
+            <Icon
+              as={StarConfetti}
+              fontSize="184px"
+              position="absolute"
+              zIndex={-1}
+              top={-8}
+              right={0}
+              transform="scaleX(-100%)"
+            />
+          </>
+        )}
+        {/* Trophy icon */}
+        <Circle
+          size="50px"
+          bg={
+            !showAnswer
+              ? "primary"
+              : currentQuestionAnswerChoice?.isCorrect
+              ? "success"
+              : "error"
+          }
+          position="absolute"
+          top={0}
+          left="50%"
+          transform="translateX(-50%) translateY(-50%)"
+        >
+          <Icon
+            as={
+              !showAnswer
+                ? Trophy
+                : currentQuestionAnswerChoice?.isCorrect
+                ? Correct
+                : Incorrect
+            }
+            fontSize="1.75rem"
+            color="neutral"
+          />
+        </Circle>
+        {quizData ? (
+          <>
+            <Center>
+              <Text fontStyle="normal" fontWeight="700" color="primaryHover">
+                {quizData.title}
+              </Text>
+            </Center>
+            {/* Progress bar */}
+            <Center gap={PROGRESS_BAR_GAP} marginBottom={6}>
+              {quizData.questions.map(({ id }, index) => {
+                /* Calculate width percent based on number of questions */
+                const width = `calc(${Math.floor(
+                  100 / quizData.questions.length
+                )}% - ${PROGRESS_BAR_GAP})`
+                return (
+                  <Container
+                    key={id}
+                    bg={progressBarBackground(index)}
+                    h="4px"
+                    w={width}
+                    maxW={`min(${width}, 2rem)`}
+                    marginInline={0}
+                    p={0}
+                  />
+                )
+              })}
+            </Center>
+            <Center>
```

Comment:
```suggestion
            <Center maxW={'29.75rem'} m='auto'>
```

This brings it in line with figma style